examples/DotNetCore/EventHub/appsettings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "EventHubConnection": {
+    "EventHubConnectionString": "",
+    "EventHubName": "demo-hub-1",
+    "BlobStorageConnectionString": "",
+    "BlobStorageContainerName": "ehcontainer",
+    "AppConfigConnectionString": ""
```

Comment:
This is a secret. You know how easy one can accidently include an actual secret value in a PR, right?